strongbox-web-core/src/test/resources/etc/conf/strongbox.xml

```diff
@@ -0,0 +1,387 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there something special about this configuration file, that is different than the standard `strongbox.xml` file? (It's a bit hard to tell, so I'm just asking for my better understanding).